network-api/networkapi/wagtailpages/migrations/0049_buyersguideproductcategoryarticlepagerelation.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.2.13 on 2022-08-10 12:03
```

Comment:
This migration is creating the relationship between Product Categories and their related articles